application/features/Audio.py

```
@@ -57,13 +60,16 @@
 
     def connect(self, *args, **kwargs):
+        logger.debug("Audio: Establishing Audio connection")
         return super().connect(*args, **kwargs)
 
     def launch_audio(self):
         try:
+            logger.debug("Audio: Launching Audio connection. Forwarding request to 127.0.0.1, port 0.")
             self.transport = self.client.get_transport()
             self.remote_port = self.transport.request_port_forward('127.0.0.1', 0)
         except Exception as e:
+            logger.warning("Audio: exception raised during launch audio: {}".format(e))
```

Ruff (0.8.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)